obspy/taup/__init__.py

```diff
@@ -185,6 +184,63 @@
     TauPyModel().get_ray_paths(
         500, 130, phase_list=["ttbasic"]).plot(plot_type="cartesian")
 
+Travel times for these ray paths can be plotted using the
+:meth:`~obspy.taup.tau.Arrivals.timeplot` method:
+
+>>> arrivals = model.get_ray_paths(source_depth_in_km=500,
+...                                distance_in_degree=130)
+>>> arrivals.timeplot()  # doctest: +SKIP
+
+.. plot::
+    :width: 50%
+    :align: center
+
+    from obspy.taup import TauPyModel
+    TauPyModel().get_ray_paths(500, 130).timeplot()
+
+Alternatively, convenience wrapper functions are in place to plot the
+travel times and the ray paths.
```

Comment:
Maybe replace with "... to plot travel times (maybe uses arrival times instead?) and ray paths across many distances."